prowler/providers/aws/lib/s3/s3.py

```diff
@@ -147,3 +150,38 @@ def send_to_bucket(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}] -- {error}"
             )
         return uploaded_objects
+
+    @staticmethod
+    def test_connection(
+        session, bucket_name: str, raise_on_exception: bool = True
```

Comment:
I think for now is good to request a session but probably we will need to setup the same credentials as the provider.

Reply:
Nice